docs/source/deployment/databricks/databricks_deployment_workflow.md

```diff
@@ -0,0 +1,311 @@
+
+### Upload project data and configuration to DBFS
+
+```{note}
+A Kedro project's configuration and data do not get included when it is packaged. They must be stored somewhere accessible to allow your packaged project to run.
+```
+
+Your packaged Kedro project needs access to data and configuration in order to run. Therefore, you will need to upload your project's data and configuration to a location accessible to Databricks. In this guide, we will store the data on the Databricks File System (DBFS).
+
+The Databricks Iris starter contains a [catalog](../../data/data_catalog.md#the-data-catalog) that is set up to access data stored in DBFS (`<project_root>/conf/`). You will point your project to use configuration stored on DBFS using the `--conf-source` option when you create your job on Databricks.
+
+There are several ways to upload data to DBFS: you can use the [DBFS API](https://learn.microsoft.com/en-us/azure/databricks/dev-tools/api/latest/dbfs), the [`dbutils` module](https://docs.databricks.com/dev-tools/databricks-utils.html) in a Databricks notebook or the [Databricks CLI](https://docs.databricks.com/dev-tools/cli/dbfs-cli.html). In this guide, it is recommended to use the Databricks CLI because of the convenience it offers.
+
+- **Upload your project's data and config**: at the command line in your local environment, use the following Databricks CLI commands to upload your project's locally stored data and configuration to DBFS:
+
+```bash
+databricks fs cp --recursive <project_root>/data/ dbfs:/FileStore/iris-databricks/data
+databricks fs cp --recursive <project_root>/conf/ dbfs:/FileStore/iris-databricks/conf
+```
+
+The `--recursive` flag ensures that the entire folder and its contents are uploaded. You can list the contents of the destination folder in DBFS using the following command:
+
+```bash
+databricks fs ls dbfs:/FileStore/iris-databricks/data
+```
+
+You should see the contents of the project's `data/` directory printed to your terminal:
+
+```bash
+01_raw
+02_intermediate
+03_primary
+04_feature
+05_model_input
+06_models
+07_model_output
+08_reporting
+```
+
+## Deploy and run your Kedro project using the workspace UI
+
+To run your packaged project on Databricks, login to your Databricks account and perform the following steps in the workspace:
+
+1. **Create a new job**: In the Databricks workspace, navigate to the `Worfklows` tab and click `Create Job`:
+
+![Create Databricks job](../../meta/images/databricks_create_new_job.png)
+
+2. **Create a new job cluster**: Create a dedicated [job cluster](https://docs.databricks.com/clusters/index.html) to run your job by clicking on the drop-down menu in the `Cluster` field:
+
+![Create Databricks job cluster](../../meta/images/databricks_create_job_cluster.png)
+
+Configure the job cluster with the following settings:
+
+- In the `name` field enter `kedro_deployment_demo`.
+- Select the radio button for `Single node`.
```

Comment:
I got confused here because, where there is an existing cluster, the UI I see is to configure the task first. Worth clarifying?

![image](https://github.com/kedro-org/kedro/assets/316517/1f1f3619-4d2c-4714-854f-ff5a4f919389)

Reply:
You're right, this is worth clarifying, as to create a cluster you have to click on the drop down. This is a good point, I will make this change.

Reply:
I added a revised image and wording to this part of the guide to make it clear that a user needs to create a new job cluster specifically for this job and shows them how to do it.